internal/proto/reader.go

```diff
@@ -4,18 +4,36 @@ import (
 	"bufio"
 	"fmt"
 	"io"
+	"math"
+	"math/big"
+	"strconv"
 
 	"github.com/go-redis/redis/v8/internal/util"
 )
 
 const (
-	ErrorReply  = '-'
-	StatusReply = '+'
-	IntReply    = ':'
-	StringReply = '$'
-	ArrayReply  = '*'
+	RespStatus    = '+' // +<string>\r\n
+	RespError     = '-' // -<string>\r\n
+	RespString    = '$' // $<length>\r\n<bytes>\r\n
+	RespInteger   = ':' // :<number>\r\n
+	RespNil       = '_' // _\r\n
+	RespFloat     = ',' // ,<floating-point-number>\r\n (golang float)
+	RespBool      = '#' // true: #t\r\n false: #f\r\n
+	RespBlobError = '!' // !<length>\r\n<bytes>\r\n
+	RespVerb      = '=' // =<length>\r\nFORMAT:<bytes>\r\n
```

Comment:
`Verb` looks like a wrong shortcut for `verbatim`. Let's keep the long version `RespVerbatim`